player/lua/select.lua

```diff
@@ -345,6 +345,59 @@ mp.add_key_binding(nil, "select-audio-device", function ()
     })
 end)
 
+mp.add_key_binding(nil, "select-history", function ()
+    local watch_later_dir = mp.get_property("current-watch-later-dir")
+
+    if not watch_later_dir then
+        show_error("No watch later files found.")
```

Comment:
This is warning, not an error. Nothing fails, just is not available.

Reply:
Every other select binding does this when nothing available though? Do you mean to change to mp.msg.warning?

Reply:
Select edition shows error only if `current-edition` is nil, but shows empty list when there are no editions available.

I just feel like, lack of files in history is not an error, so `mp.msg.warning`. Maybe other indeed could be changed like that. Generally OSD message is enough, but we show also terminal message, errors are scary.

Reply:
That's not intended. I wrote it blindly and thought no current-edtion means there are no editions like no chapter means there are no chapters in the previous binding.